### DISCUSSION TITLE
Cannot find web gui after installation

### DISCUSSION THREAD

**Discussion by vesta-james-rosser** [Security-Onion-Solutions/securityonion#13358] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Installed from iso initially as if had access to the inet, but it failed. I reinstalled as airgap and I cannot reach the console. Our firewalls are not the issue, but the IP's used are not working, have I missed something?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Have you tried using the so-firewall command as shown in the documentation?
https://docs.securityonion.net/en/2.4/post-installation.html#adjust-firewall-rules
